src/settings/permissions/PermissionSetDetails.js

```diff
@@ -122,6 +125,7 @@ class PermissionSetDetails extends React.Component {
           onToggle={this.handleSectionToggle}
           expanded={sections.assignedUsers}
           permissionsSet={selectedSet}
+          tenantId={stripes.okapi.tenant}
```

Comment:
probably you miss how consortia manager works, https://github.com/folio-org/ui-consortia-settings/blob/master/src/routes/ConsortiumManager/settings/users/general/PermissionSets/PermissionSets.js#L137C37-L137C37

`stripes.okapi.tenant` won't have selected tenant in stripes.okapi, stripes.okapi has only active tenant